### ISSUE TITLE
Transfering Sensu mattermost plugin 

### ISSUE THREAD

**Issue by mattronix** [sensu-plugins/community#118]
## Transfer Ownership

We would like to transfer the ownership of this plugin to the sensu-plugins organization in hopes of better community support and exposure. I have reviewed [the skeleton plugin](https://github.com/sensu-plugins/sensu-plugins-skel) for best practices.

Repo to transfer: https://github.com/mattronix/handler-mattermost

I am also interested in continuing to help as a maintainer of this plugin 👍

**Comment by majormoses** [comment]
Happy to help facilitate moving this to the community. If you could please read through [this](https://github.com/sensu-plugins/community/blob/master/TRANSFERRING_REPOS.md) and follow the process to transfer it to myself I will get it transferred to the organization.

**Comment by majormoses** [comment]
Also I noticed that its licensed under GPv3 can I have your consent upon transferring it to the community to switch it to the MIT license that sensu uses?

**Comment by majormoses** [comment]
@mattronix have not heard back from you, did you want to get this going still?

**Comment by mattronix** [comment]
Yes i do sorry for my delayed responses hectic times what do i need to do ?


Sent from my Samsung Galaxy smartphone.<div>
</div><div>
</div><div>-------- Original message --------</div><div>From: Ben Abrams <notifications@github.com> </div><div>Date: 09/09/2018  01:15  (GMT+01:00) </div><div>To: sensu-plugins/community <community@noreply.github.com> </div><div>Cc: Matthew Frost <m.frost@mattronix.nl>, Mention <mention@noreply.github.com> </div><div>Subject: Re: [sensu-plugins/community] Transfering Sensu mattermost plugin&nbsp; (#118) </div><div>
</div>>

**Comment by majormoses** [comment]
The process is outlined here: https://github.com/sensu-plugins/community/blob/master/TRANSFERRING_REPOS.md

**Comment by mattronix** [comment]
Step 4 and onwards is left?


Sent from my Samsung Galaxy smartphone.<div>
</div><div>
</div><div>-------- Original message --------</div><div>From: Ben Abrams <notifications@github.com> </div><div>Date: 09/09/2018  03:44  (GMT+01:00) </div><div>To: sensu-plugins/community <community@noreply.github.com> </div><div>Cc: Matthew Frost <m.frost@mattronix.nl>, Mention <mention@noreply.github.com> </div><div>Subject: Re: [sensu-plugins/community] Transfering Sensu mattermost plugin&nbsp; (#118) </div><div>
</div>>

**Comment by majormoses** [comment]
Ideally we would start back up  at step 3 but we can always do that post transfer.

**Comment by mattronix** [comment]
Clear will start in 20 mins


Sent from my Samsung Galaxy smartphone.<div>
</div><div>
</div><div>-------- Original message --------</div><div>From: Ben Abrams <notifications@github.com> </div><div>Date: 09/09/2018  19:47  (GMT+01:00) </div><div>To: sensu-plugins/community <community@noreply.github.com> </div><div>Cc: Matthew Frost <m.frost@mattronix.nl>, Mention <mention@noreply.github.com> </div><div>Subject: Re: [sensu-plugins/community] Transfering Sensu mattermost plugin&nbsp; (#118) </div><div>
</div>>

**Comment by mattronix** [comment]
Do I transfer this to you?


On 9 September 2018 at 19:47:54, Ben Abrams (notifications@github.com) wrote:

Ideally we would start a step 3 but we can always do that post transfer.

**Comment by majormoses** [comment]
yes, I can transfer it to the org. The org is setup to only allow admins of the org to transfer a repo in.

**Comment by mattronix** [comment]
Can you give me the org name just to confirm.


Sent from my Samsung Galaxy smartphone.<div>
</div><div>
</div><div>-------- Original message --------</div><div>From: Ben Abrams <notifications@github.com> </div><div>Date: 14/09/2018  17:35  (GMT+01:00) </div><div>To: sensu-plugins/community <community@noreply.github.com> </div><div>Cc: Matthew Frost <m.frost@mattronix.nl>, Mention <mention@noreply.github.com> </div><div>Subject: Re: [sensu-plugins/community] Transfering Sensu mattermost plugin&nbsp; (#118) </div><div>
</div>>

**Comment by majormoses** [comment]
`sensu-plugins` or you can transfer it to it to me you can use `majormoses`

https://github.com/sensu-plugins

**Comment by mattronix** [comment]
Transfered to you :)


Sent from my Samsung Galaxy smartphone.<div>
</div><div>
</div><div>-------- Original message --------</div><div>From: Ben Abrams <notifications@github.com> </div><div>Date: 14/09/2018  22:24  (GMT+01:00) </div><div>To: sensu-plugins/community <community@noreply.github.com> </div><div>Cc: Matthew Frost <m.frost@mattronix.nl>, Mention <mention@noreply.github.com> </div><div>Subject: Re: [sensu-plugins/community] Transfering Sensu mattermost plugin&nbsp; (#118) </div><div>
</div>>

**Comment by majormoses** [comment]
I have transfered it to the org: https://github.com/sensu-plugins/handler-mattermost

I will probably this weekend rename the repo to match out conventions and work on getting the basic skel, ci, cd, etc setup.

**Comment by majormoses** [comment]
I have created https://github.com/sensu-plugins/handler-mattermost/issues/3 to track its prep for being maintained.

**Comment by mattronix** [comment]
Thank you i hope people find it useful :)


Sent from my Samsung Galaxy smartphone.<div>
</div><div>
</div><div>-------- Original message --------</div><div>From: Ben Abrams <notifications@github.com> </div><div>Date: 14/09/2018  23:20  (GMT+01:00) </div><div>To: sensu-plugins/community <community@noreply.github.com> </div><div>Cc: Matthew Frost <m.frost@mattronix.nl>, Mention <mention@noreply.github.com> </div><div>Subject: Re: [sensu-plugins/community] Transfering Sensu mattermost plugin&nbsp; (#118) </div><div>
</div>>

**Comment by majormoses** [comment]
I am sure people will, I was looking for a mattermost handler myself for a consulting gig a while back and they did not want to pay for me to develop one from scratch.

**Comment by mattronix** [comment]
Lovely :)


Sent from my Samsung Galaxy smartphone.<div>
</div><div>
</div><div>-------- Original message --------</div><div>From: Ben Abrams <notifications@github.com> </div><div>Date: 15/09/2018  18:50  (GMT+01:00) </div><div>To: sensu-plugins/community <community@noreply.github.com> </div><div>Cc: Matthew Frost <m.frost@mattronix.nl>, Mention <mention@noreply.github.com> </div><div>Subject: Re: [sensu-plugins/community] Transfering Sensu mattermost plugin&nbsp; (#118) </div><div>
</div>>
